common/v2/features/ScreenLock/ScreenLockLocking.tsx

```diff
@@ -104,3 +104,5 @@ export default class ScreenLockLocking extends Component<LockScreenProps> {
     );
   }
 }
+
+export default ScreenLockLocking;
```

Comment:
Is there an advantage over exporting the class separately, rather than directly from the line where the class is declared?

Reply:
Good catch. I think I used it for testing an HOC and forgot to change it back